src/components/Colors/Colors.stories.tsx

```diff
@@ -0,0 +1,254 @@
+import * as React from "react";
+
+import { boolean, text } from "@storybook/addon-knobs";
+import cssVariables from "../../helpers/CSSVariablesHelper";
+import getCSSVariable from "../../helpers/getCSSVariable";
+import UIDocCard from "./UIDocCard";
+import Card from "../Card/Card";
+import List from "../List/List";
+import Heading from "../Heading/Heading";
+import { ListTypes } from "../List/ListTypes";
+
+export default {
+    title: "Colors",
+    decorators: [(storyFn) => <div style={{ padding: "2%" }}>{storyFn()}</div>],
+};
+
+const uiVariables: { [k: string]: string } = {};
```

Comment:
`{ [k: string]: string }` comes up a lot throughout this PR. It'd be good to have this as a typescript `interface` probably in `/src/interfaces.js` and import it wherever you need it.